### ISSUE TITLE
Freelancer: Signup template does not support extended user fields. 

### ISSUE THREAD

**Issue by CaMer0n** [Jimmi08/e107-free-themes-by-e107sk#1]
### Partial Fix: 

Add this to the template file: 
```
if(!isset($SIGNUP_EXTENDED_USER_FIELDS))
{
	$SIGNUP_EXTENDED_USER_FIELDS	= "
	<div class='control-group'>
	<div class='form-group floating-label-form-group controls mb-0 pb-2'>
	    <label>{EXTENDED_USER_FIELD_TEXT}</label>
			{EXTENDED_USER_FIELD_EDIT}
		</div>
		</div>";
}
```

Add the following to `$SIGNUP_BODY`
```
{SIGNUP_EXTENDED_USER_FIELDS: class=form-control rounded-0&placeholder=[label]}
```

### Expected Result Example: 

![image](https://user-images.githubusercontent.com/2825679/79800419-05033380-8311-11ea-90ed-8b06a07e7810.png)

**Comment by Jimmi08** [comment]
@CaMer0n thanks, the theme will be revised after 2.3 release

**Comment by Jimmi08** [comment]
Fixed in version 4.1.

![image](https://user-images.githubusercontent.com/5429548/80871383-9331cf00-8cac-11ea-9544-6a8dbee59f2b.png)
